### PR TITLE
Darkmode rework

### PR DESCRIPTION
**Pull request by ameliadd**
Here's the latest version of the darkmode rework. With the new colors in the buttons (including the light mode of the Script, Flows and Apps buttons color rework) and many other smaller elements.

### CODE REVIEW

File: frontend/src/lib/components/apps/editor/contextPanel/components/OutputHeader.svelte [hunk 1]
```diff
@@ -39,12 +39,12 @@
...
 		blue: 'bg-blue-50 dark:bg-frost-800',
-		indigo: 'bg-indigo-50 dark:bg-indigo-800'
+		indigo: 'bg-indigo-50 dark:bg-indigo-700/50'
 	}
 
 	const manuallyOpenColor = {
-		blue: 'text-primary bg-gray-300 rounded-sm',
-		indigo: 'text-primary bg-gray-300 rounded-sm'
+		blue: 'text-primary bg-gray-300 rounded-sm dark:bg-gray-600',
+		indigo: 'text-primary bg-gray-300 rounded-sm dark:bg-gray-600'
```
Comment (fatonramadani) [inline frontend/src/lib/components/apps/editor/contextPanel/components/OutputHeader.svelte]: The text is not that readable:
Screenshot 2024-04-19 at 17 08 23

File: frontend/src/lib/components/common/alert/Alert.svelte [hunk 2]
```diff
@@ -40,28 +40,28 @@
...
 	const classes: Record<AlertType, Record<string, string>> = {
 		info: {
-			bgClass: 'bg-blue-50 border-blue-200 border dark:bg-blue-900/50 dark:border-blue-900',
-			iconClass: 'text-blue-500 dark:text-blue-400',
-			titleClass: 'text-blue-800 dark:text-blue-100',
-			descriptionClass: 'text-blue-700 dark:text-blue-200'
+			bgClass: 'bg-blue-50 border-blue-200 border dark:bg-blue-900/40 dark:border-blue-700/40',
+			iconClass: 'text-blue-500 dark:text-blue-300/90',
+			titleClass: 'text-blue-800 dark:text-blue-100/90',
+			descriptionClass: 'text-blue-700 dark:text-blue-100/90'
```
Comment (fatonramadani) [inline frontend/src/lib/components/common/alert/Alert.svelte]: Screenshot 2024-04-19 at 17 09 54
Screenshot 2024-04-19 at 17 09 57
I'm ok with the icon color change, but the text looks quite dimmed, wdyt?

Comment (ameliadd) [reply frontend/src/lib/components/common/alert/Alert.svelte]: Ohh, you're right. Could keep it at 100% opacity

File: frontend/src/lib/components/apps/editor/contextPanel/components/OutputHeader.svelte (hunk 1 above)
Comment (ameliadd) [reply frontend/src/lib/components/apps/editor/contextPanel/components/OutputHeader.svelte]: the "ctx"? Could be in grey-50

**Pull request merged**